typedoc.js

```diff
@@ -1,29 +1,9 @@
-const fs = require('fs');
-const path = require('path');
-
-function getDirectories(path) {
-  return fs.readdirSync(path).filter(function (file) {
-    return fs.statSync(path + '/' + file).isDirectory();
-  });
-}
-
-const packages = getDirectories(path.join(__dirname, 'packages'));
-
-const entryPoints = packages
-  .flatMap(p => [`packages/${p}/src/index.ts`, `packages/${p}/src/index.tsx`])
-  .filter(function (path) {
-    return fs.existsSync(path);
-  });
-
-const exclude = packages
-  .flatMap(p => [`packages/${p}/tests`])
-  .concat('**/node_modules/**');
-
 module.exports = {
-  entryPoints,
-  exclude,
+  entryPoints: '.',
+  entryPointStrategy: 'packages',
+  exclude: ['examples/**', '**/tests/**', 'packages/default-theme'],
+  includeVersion: true,
   name: '@lumino',
   out: 'docs/source/api',
-  readme: 'README.md',
-  tsconfig: 'tsconfigdoc.json'
+  readme: 'none'
```

Comment:
Setting `readme` to `none` means that the index page shows the list of packages rather than the repo readme, which allows us to get rid of the index-copy step in the conf.py file.

Reply:
I found the package first page for this PR: https://lumino--354.org.readthedocs.build/en/354/api/modules/algorithm.html
much better than the current one: https://lumino.readthedocs.io/en/stable/api/algorithm/index.html

So definitely :100: for this new setup